### PR TITLE
i18n deployment backend & lang.js sort function

### PR DESCRIPTION
**Pull request by BennieMeng**
**What type of PR is this?**
> Uncomment only one, leave it on its own line:
>
> /kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

```release-note
 backend deployment i18n
 sort function lang.js
```

### CODE REVIEW

File: src/frontend/src/app/admin/cluster/trash-cluster/trash-cluster.component.html [hunk 1]
```diff
@@ -36,8 +36,8 @@ <h2 class="header-title">已删除集群列表</h2>
       </clr-dg-column>
       <clr-dg-row *ngFor="let cluster of clusters" [clrDgItem]="cluster">
         <clr-dg-action-overflow>
-          <button class="action-item" (click)="recoverCluster(cluster)">恢复</button>
-          <button class="action-item" (click)="deleteCluster(cluster)">彻底删除</button>
+          <button class="action-item" (click)="recoverCluster(cluster)">{{'BUTTON.RESTORE' | translate}}</button>
+          <button class="action-item" (click)="deleteCluster(cluster)">{{'BUTTON.DELETE_COMPLETE' | translate}}</button>
```
Comment (wilhelmguo) [inline src/frontend/src/app/admin/cluster/trash-cluster/trash-cluster.component.html]: 感觉这个用Delete Permanently 比较确切

Comment (BennieMeng) [reply src/frontend/src/app/admin/cluster/trash-cluster/trash-cluster.component.html]: done

**Comment by wilhelmguo** [comment]
@BennieMeng 已经修改的问题或者不需要修改的问题点击这个标记为已解决 
image

File: src/frontend/lang.js [hunk 2]
```diff
@@ -62,4 +62,46 @@ if (process.argv[2] === 'format') {
       console.log('done');
     })
   }
+}
+
+// sort 不传数据会 sort 所有文件，或者换紧跟文件前缀
```
[the remark this answers is not in the document]
Comment (BennieMeng) [reply src/frontend/lang.js]: 这块在 wiki 中解释吧

File: src/frontend/lang.js [hunk 3]
```diff
@@ -62,4 +62,46 @@ if (process.argv[2] === 'format') {
...
+      } else {
+        files.forEach(function (filePath) {
+          fs.writeFile(path.resolve(dirPath, filePath), JSON.stringify(sort(require(path.resolve(dirPath, filePath))), null, 2), 'utf8', err => {
+            if (err) throw error(err);
+            console.log(filePath + ' done');
+          })
+        })
+      }
+    })
+  }
```
[the remark this answers is not in the document]
Comment (BennieMeng) [reply src/frontend/lang.js]: 这个在如果需要添加新的脚本时候可以归类。

[the remark this answers is not in the document]
Comment (BennieMeng) [reply src/frontend/lang.js]: 用法可以查看wiki，我可以在文件中加上地址。

**Comment by BennieMeng** [comment]
format lang.js，pls have a look @olivewind

**Comment by olivewind** [comment]
> format lang.js，pls have a look @olivewind

good job

**Pull request merged**